src/index.html

```diff
@@ -5,8 +5,305 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo_img">
+        <img src="https://github.com/mate-academy/layout_moyo-header/blob/master/src/images/logo.png?raw=true" alt="Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#Apple"
+              class="nav__link is-active"
+              data-qa="hover"
+             >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item" >
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="nav__link"
+            href="#Laptops"
+            data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"  href="#gadgets">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="container">
+      <div class="card" data-qa="card">
+        <div class="card__image-wrapper">
+          <img class="card__img" src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+        </div>
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="product-code">Product code: 195434</p>
+
+        <div class="reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews__text">Reviews: 5</p>
+
+        </div>
+        <div class="price">
+
+          <p class="price_label">Price:</p>
+
+          <p class="price_value">$2,199</p>
+        </div>
```

Comment:
Need to fix it everywhere
```suggestion
        <div class="price">
          <p class="price_label">Price:</p>
          <p class="price_value">$2,199</p>
        </div>
```